site/content/get-started.md

```diff
@@ -201,13 +208,14 @@ The YAML code in the following sections can be found in the [cafe-example folder
 
 Create the file _cafe.yaml_ with the following contents:
 
-{{< ghcode "https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/refs/heads/main/examples/cafe-example/cafe.yaml">}}
+{{< ghcode `https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/refs/heads/main/examples/cafe-example/cafe.yaml`>}}
 
-Apply it:
+Apply it using `kubectl`:
```

Comment:
Don't think we need this change

Reply:
Just added it to be consistent with the other statements [below](https://github.com/nginxinc/nginx-gateway-fabric/blob/main/site/content/get-started.md?plain=1#L237).